index-browser.js

```diff
@@ -1,3 +1,2 @@
-const level = require('level-js')
```

Comment:
Seems these files can be deleted and replaced with a single index file

Reply:
Done and done!